api/v1alpha3/types.go

```diff
@@ -554,6 +554,9 @@ type Instance struct {
 
 	// The tags associated with the instance.
 	Tags map[string]string `json:"tags,omitempty"`
+
```

Comment:
The core machine and the provider API already have semantic for this -> `.failureDomain`. See https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/425e1bb337cfb76ceb38c1cce97345c26fafe08d/pkg/cloud/services/ec2/instances.go#L162-L165